tutorials/graph-theory/bellman-ford-algorithm.md

```diff
@@ -35,4 +35,20 @@ void bellman_ford(T_a3 &g, T_vector &dist, int src, int mx_edges) {
 }
 ```
 </TabItem>
-</Tabs>
\ No newline at end of file
+
+<TabItem value="py" label="Python">
+<SolutionAuthor name="@Odion-Sonny"/>
+
+```py
+def bellman_ford(g, dist, src, mx_edges):
```

Comment:
unlike c++, this function won't work as `dist` would stay in the scope of the function. You need to add the return statement.

```py
return dist
```

A simple verification:

```py
class Solution:
    def findCheapestPrice(self, n: int, flights: List[List[int]], src: int, dst: int, k: int) -> int:
        def bellman_ford(g, dist, src, mx_edges):
            dist[src] = 0
            for i in range(mx_edges + 1):
                ndist = dist.copy()
                for x in g:
                    _from, to, cost = x
                    ndist[to] = min(ndist[to], dist[_from] + cost)
                dist = ndist
            return dist
        g = []
        for f in flights: g.append([f[0], f[1], f[2]])
        cost = bellman_ford(g, [1e9] * n, src, k)
        if cost[dst] == 1e9: return -1
        return cost[dst]
```

Reply:
Thanks for pointing this out, that was a silly oversight.